src/BrazeKit-dev.js

```diff
@@ -512,6 +592,24 @@ var constructor = function() {
     this.onUserIdentified = onUserIdentified;
     this.removeUserAttribute = removeUserAttribute;
     this.decodeClusterSetting = decodeClusterSetting;
+
+    function kitLogger(method) {
```

Comment:
Feels like a nitpick, but I would also wrap a small tests around this logger, in case we break it in the future.

Reply:
updated.